src/colvar.cpp

```diff
@@ -1238,10 +1245,16 @@ int colvar::init_dependencies() {
     init_feature(f_cv_linear, "linear", f_type_static);
     init_feature(f_cv_homogeneous, "homogeneous", f_type_static);
 
-    // because total forces are obtained from the previous time step,
-    // we cannot (currently) have colvar values and total forces for the same timestep
     init_feature(f_cv_multiple_ts, "multiple_timestep", f_type_static);
-    exclude_feature_self(f_cv_multiple_ts, f_cv_total_force_calc);
+
+    // when total atomic forces are obtained from the previous time step,
+    // we cannot (currently) have colvar values and projected total forces for the same timestep
+    // TODO this will need refining for driven alchemical parameters
```

Comment:
What do you mean by "driven alchemical parameters"?

Reply:
"Driven" means Colvars is responsible for integrating (driving) the dynamics of this parameter. For the current version of this, see  https://github.com/Colvars/colvars/blob/5e37f65d764c38eec892fadd753095747eeb37d1/src/colvar.cpp#L332